scripts/test.sh

```diff
@@ -10,14 +10,14 @@ pytest --verbose --cov zoloto/ --cov-report term --cov-report html tests/
 if hash black 2>/dev/null;
 then
     echo "> Running formatter..."
-    black zoloto tests benchmarks examples setup.py docs --check
+    black zoloto tests benchmarks examples setup.py docs stubs --check
 fi
 
 echo "> Running linter..."
-flake8 zoloto tests benchmarks examples setup.py docs
+flake8 zoloto tests benchmarks examples setup.py docs stubs
 
 echo "> Running isort..."
-isort -rc -c zoloto tests benchmarks examples setup.py docs
+isort -rc -c zoloto tests benchmarks examples setup.py docs stubs
 
 echo "> Running type checker..."
 mypy zoloto tests examples docs
```

Comment:
Should `mypy` not be run on the stubs?

Reply:
d9f34f3

No hard in running it I guess.